api/src/main/proto/stellarstation/api/v1/antenna/antenna.proto

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package stellarstation.api.v1.antenna;
+
+option go_package = "github.com/infostellarinc/go-stellarstation/api/v1/antenna";
+
+option java_multiple_files = true;
+option java_outer_classname = "AntennaProto";
+option java_package = "com.stellarstation.api.v1.antenna";
+
+// Messages for modeling physical antenna configuration in the API.
+
+// The static configuration of an antenna. These are the physical parameters of the antenna.
+message AntennaConfiguration {
+
+  // The latitude of the antenna, in degrees.
+  double latitude = 1;
+
+  // The longitude of the antenna, in degrees.
+  double longitude = 2;
+
+  // The altitude of the antenna, in meters.
+  double altitude_meters = 3;
+
+  // The heading (i.e., left-right rotation), in degrees.
+  // https://en.wikipedia.org/wiki/Euler_angles#Tait-Bryan_angles
+  double heading_degrees = 4;
```

Comment:
@vaelen What do you think? I think we may just want to remove the rotation fields, I have a feeling they wouldn't be very standard across ground stations.

Reply:
They are values that the radio's monitoring API returns and we're erring on including everything the radio returns in out monitoring API too. But if you think it's too out there we should remove it. What do you think?

Reply:
ACU's can fine track a satellite using the information on the electromagnetic wave, regardless of the TLE files. This specific ACU is estimating the attitude of the satellite using the incoming wave.

Would you please mind referring me to the specific section of the manual where these numbers are defined?

Reply:
Just FYI: I found this when I was reading manual of another MCS.
https://drive.google.com/drive/u/0/folders/1_RuFaq6L42giRT5sws4rpZH1ueTgnUJ_

page 13 of Software ICD V7.1.pdf
> The <station> element has the following properties:
 name: The name of the station.
 latitudeDeg: The station latitude in degrees north (southerly latitudes are negative).
 longitudeDeg: The station longitude in degrees east (westerly longitudes are negative).
 altitudeM: The station altitude (above sea level) in meters.
 __headingDeg: The station 'heading' in degrees (the physical orientation of the pedestal).__
 (optional) isDefault: 1 if this station is the default station (selected when the session is
loaded), 0 otherwise

This software only reports heading from those three angles and I found the description too ambiguous.

Reply:
In accordance with the available documentation, these values are the estimation of the attitude of the satellite, when the MCS is configured to work in an automatic mode called "Attitude". @vaelen is going to check with the provider for documentation with specific definitions.

Anyhow, @anuraaga : my recommendation is that we remove these values from the API since they are very specific for this device.